.github/workflows/go.yml

```diff
@@ -81,6 +81,21 @@ jobs:
     - name: Test registry server with PostgreSQL
       run: go test ./server/registry -postgresql
 
+    - name: Test a remote (standalone) registry server in hosted mode
+      env:
+        APG_REGISTRY_ADDRESS: localhost:8080 
+        APG_REGISTRY_INSECURE: 1
+        PROJECTID: hosted
```

Comment:
Let's pick a project name that's more obviously for test purposes. Something like `hosted-ci-test`.

Reply:
Done. `hosted-ci-test`